src/stories/molecules/GrafHeader/GrafHeader.style.ts

```diff
@@ -16,7 +16,7 @@ export const GrafHeaderWrapper = styled.div`
 
 export const TextContent = styled.div`
   display: flex;
-  flex-direction: column;
+  flex-direction: 'column';
```

Comment:
Is this the correct syntax? I've never seen this in quotes before.

Reply:
This was me not being careful. Lesson learnt!